crud/common/schema.lua

```diff
@@ -222,13 +224,14 @@ end
 -- `{res = ..., err = ..., space_schema_hash = ...}`
 -- space_schema_hash is computed if function failed and
 -- `add_space_schema_hash` is true
-function schema.wrap_box_space_func_result(space, box_space_func_name, box_space_func_args, opts)
-    dev_checks('table', 'string', 'table', 'table')
+function schema.wrap_box_space_func_result(conn, space_name, box_space_func_name, box_space_func_args, opts)
```

Comment:
word `box` states that conn is always a `box`. Please remove this change and inline box instead of conn where it needed.

Reply:
reverted